propagation/toolboxcloud.md

```diff
@@ -1,10 +1,25 @@
+The toolbox cloudification service makes available single data processing (not visualization) functions of existing toolboxes on APEx. These data processing operations can then be freely combined with other algorithms hosted on APEx. Instead of a file-centric and local processing scheme of a traditional toolbox, key operations can be performed in the cloud where EO data archives are directly available as data cubes, where arbitrary extends in time and space can be extracted. Furthermore, functions from various toolboxes can be combined in a single workflow through APEx. 
 
-The process begins with an assessment of your request to estimate the effort required to cloudify the toolbox. The main
-advantage of this approach is that the specific functionality will also become available as a modular building block for
-other projects. Therefore, it is crucial to provide a compelling justification for why this feature deserves to be 'cloudified'.
+Service users may select functionalities from a toolbox that they would like to use on the APEx platform. The proposed functionalities and their toolbox will then be reviewed by ESA and APEx experts. Similar to the algorithm enhancement service, an effort estimation and an offer will be made following the review. On acceptance of the offer, the requested functionalities will be provided as an Application Package according to OGC best practice, or as an openEO User Defined Function or Process (UDF/UDP). 
 
+To ensure interoperability between procedures from different toolboxes, it is necessary to share data using standard data formats, writing intermediary files to disk. Users should consider if the flexibility of this approach outweighs the performance drawbacks and request deployment as a single, integrated package instead, if performance is critical. 
+
+Functionalities made available through the toolbox cloudification service are hosted via the APEx algorithm hosting service. A similar set of features is offered by the algorithm enhancement service, which focuses on improving maturity and performance of algorithms and preparing them for hosting on APEx. The algorithm enhancement service works together with scientists and algorithm developers to make their algorithms available on APEx. 
+
+#### Features of the Toolbox Cloudification Service 
+
+The toolbox cloudification service offers an analysis of the implementation of the toolbox and the requested processing capability. On the basis of this analysis, an implementation strategy is proposed and executed to provide the functionality outside the context of the toolbox in a fully cloud-integrated format on the APEx platform. 
+
+Projects may benefit from the toolbox cloudification service by bringing key functions from traditional desktop EO toolboxes into the cloud, where they can be integrated into large-scale processing workflows. 
+
+Toolboxes evolve over time. If a toolbox migrated to APEx, it shall be kept up to date. This requires maintenance. 
```

Comment:
How will these changes be handled? This requires maintenance does not provide much detail on who will do this, it is done automatically or upon request, through open source contributions or paid efforts, ...

Reply:
The process for maintenance is an open question for us as well.
This should be considered in the next phase of the apex project.
A similar issue exists for the algorithm enhancement service. We wrote in the introduction of the enhancement service:

> [...] and algorithm providers are encouraged to use it as a basis for further development of the algorithm.

Reply:
OK, maybe we should clarify this part a bit more in the text that this is still under investigation or even remove the text for now? Just to prevent any additional questions.

Reply:
I added a sentence to make clear that the maintenance strategy is not yet defined. It may be out of scope of APEx.
Do you think the current version is clear enough?